deploy/csi-hostpath-plugin.yaml

```diff
@@ -70,6 +70,30 @@ spec:
             - "--v=4"
             - "--endpoint=$(CSI_ENDPOINT)"
             - "--nodeid=$(KUBE_NODE_NAME)"
+            - --ignorenamespace=openshift-machine-api
+            - --ignorenamespace=openshift-kube-apiserver
+            - --ignorenamespace=openshift-kube-apiserver-operator
+            - --ignorenamespace=openshift-kube-scheduler
+            - --ignorenamespace=openshift-kube-controller-manager
+            - --ignorenamespace=openshift-kube-controller-manager-operator
+            - --ignorenamespace=openshift-kube-scheduler-operator
+            - --ignorenamespace=openshift-console-operator
+            - --ignorenamespace=openshift-controller-manager
+            - --ignorenamespace=openshift-controller-manager-operator
+            - --ignorenamespace=openshift-cloud-credential-operator
+            - --ignorenamespace=openshift-authentication-operator
+            - --ignorenamespace=openshift-service-ca
+            - --ignorenamespace=openshift-kube-storage-version-migrator-operator
+            - --ignorenamespace=openshift-config-operator
+            - --ignorenamespace=openshift-etcd-operator
+            - --ignorenamespace=openshift-apiserver-operator
+            - --ignorenamespace=openshift-cluster-csi-drivers
+            - --ignorenamespace=openshift-cluster-storage-operator
+            - --ignorenamespace=openshift-cluster-version
+            - --ignorenamespace=openshift-image-registry
+            - --ignorenamespace=openshift-machine-config-operator
+            - --ignorenamespace=openshift-sdn
+            - --ignorenamespace=openshift-service-ca-operator
```

Comment:
Is this list going to have to get updated every time a new openshift-* namespace comes along? Couldn't we exclude all openshift-* namespaces with a regexp in the code somewhere if that is what needs to be done?

Reply:
For the rotating cert scenario we may need to watch certain openshift
namespaces

I at least am fine with deferring more sophisticated regex / other means of
expression in a separate PR

Reply:
I also think we should take this in a standalone pull-request, I would like discuss this requirement further.